src/api/public/apidocs-new/OBS-v2.10.50.yaml

```diff
@@ -24,13 +25,22 @@ paths:
 
   /architectures:
     $ref: 'paths/architectures.yaml'
-
   /architectures/{architecture_name}:
     $ref: 'paths/architectures_architecture_name.yaml'
 
+  /attribute:
+    $ref: 'paths/attribute.yaml'
+  /attribute/{namespace}:
+    $ref: 'paths/attribute_namespace.yaml'
+  /attribute/{namespace}/_meta:
+    $ref: 'paths/attribute_namespace_meta.yaml'
+  /attribute/{namespace}/{attribute_name}:
```

Comment:
Can we separate namespaces and attributes? Would make this easier to understand.

Reply:
Sure. I will split them.

Reply:
Done. Now it looks like this:

![Screenshot from 2021-01-11 14-26-32](https://user-images.githubusercontent.com/24919/104188144-12b99580-5419-11eb-8193-1637934babc6.png)